node/tests/UnitTests.test.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import { describe, expect, it } from "@jest/globals";
+import { compareMaps } from "./TestUtilities";
+
+describe("test compareMaps", () => {
+    it("should compare two empty maps", () => {
```

Comment:
"should compare " doesn't imply what the test checks for - should it be equal? not equal? instead write something like "Map comparison of empty maps should return true"